packages/compass-aggregations/src/stores/store.js

```diff
@@ -249,6 +251,8 @@ const configureStore = (options = {}) => {
     );
   }
 
+  // options.isAtlasDeployed is only used by mms to change some behaviour in the
```

Comment:
As part of the above I decided to typescript this file so we can start to capture which options these stores can even take, but yeah.. this file is only the interface for mms whereas for us the plugin stuff in `packages/compass-collection/src/stores/context.tsx` is more relevant but even that gets called in `packages/compass-collection/src/modules/tabs.ts` which is called in `packages/compass-collection/src/stores/index.ts` when the 'select-namespace' event arrives but really all the options are actually contained in that `metadata` parameter and that metadata param gets constructed in at least two places..

Figured that kind of work should just be in a separate PR.

Reply:
Amazing effort, thank you for documenting this in the code!